sklearn/metrics/scorer.py

```diff
@@ -191,9 +203,11 @@ def get_scorer(scoring):
         try:
             scorer = SCORERS[scoring]
         except KeyError:
+            scorers = [scorer for scorer in SCORERS
```

Comment:
I take this as an invitation to create an issue for this instead of fixing it here ok? 😁